src/ESPEasy-Globals.h

```diff
@@ -1104,13 +1104,16 @@ struct NodeStruct
 struct systemTimerStruct
 {
   systemTimerStruct() :
-    timer(0), plugin(0), Par1(0), Par2(0), Par3(0) {}
+    timer(0), plugin(0), TaskIndex(-1), Par1(0), Par2(0), Par3(0), Par4(0), Par5(0) {}
 
   unsigned long timer;
   byte plugin;
-  byte Par1;
-  byte Par2;
-  byte Par3;
+  int16_t TaskIndex;
```

Comment:
Why change the Par1...Par5 from `byte` to `int`?
And why is TaskIndex an 16-bit int?
We currently have 12 tasks for ESP8266 and 32 tasks for ESP32.

Reply:
because to maintain compatibility with the past I needed a special case, I thought I would use the value -1.

Reply:
That makes sense :)